OpenOversight/app/templates/partials/links_and_videos_row.html

```diff
@@ -0,0 +1,32 @@
+<div class='row'>
+	{% for type, list in obj.links|groupby('link_type') %}
+		{% if type == 'link' %}
+		<div class="col-sm-12 col-md-6">
+			<h5>Links</h5>
+			<ul class='list-group'>
+				{% for link in list %}
+					<li class='list-group-item'>
+						<a href="{{ link.url }}">{{ link.url }}</a>
+					</li>
+				{% endfor %}
+			</ul>
+		</div>
+		{% endif %}
+		{% if type == 'video' %}
+			<div class="col-sm-12 col-md-6">
+				<ul class='list-group'>
+					<h5>Videos</h5>
+					{% for link in list %}
+						{% with link_url = link.url.split('v=')[1] %}
+							<li class='list-group-item'>
+								<div class='video-container'>
+									<iframe width="560" height="315" src="https://www.youtube.com/embed/{{ link_url }}" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
I do think we want to enable links to other videos - but they don't necessarily need to be embedded in non-Youtube cases - we just want to have "Videos" in a separate area together on the incident page (away from other link types, e.g. media articles)

Reply:
Cool. I'll add a 'non YouTube video' link type, that way they can be with the videos but not embedded.